setup.py

```diff
@@ -78,6 +78,7 @@
     # Execute `pip install .[test]` once and the use `pytest` to run tests
     'test': [
         'flake8',
+        'imagehash',
```

Comment:
I would add this under the `develop` (development dependencies) extra instead of the `test` (dependencies required to run the unit tests) extra

Reply:
Thanks @bouweandela I had interpreted testing more broadly than this. Now moved (b90431e), and also have proposde a tweak to the comments to avoid similar misunderstanding (036a9f62).